### PR TITLE
Fix a problem with hijackers in the survival maps

### PR DESCRIPTION
**Pull request by abcdefg30**
As reported by @Snurre86 in IRC, tanks kept moving to their waypoints/starting
a hunt activity even after you captured them with a hijacker.

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1: looks good.

File: mods/ra/maps/survival02/survival02.lua [hunk 1]
```diff
@@ -310,6 +318,9 @@ SetupSoviets = function()
 			GuardHarvester(unit, Harvester1)
```
Comment (obrakmann) [inline mods/ra/maps/survival02/survival02.lua]: The harvester guards need a `Trigger.ClearAll` too (after line 53 probably).

**Comment by obrakmann** [comment]
Aside from that one thing, :+1:

**Comment by abcdefg30** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/20629033923fbb7f7ac975355b702e797312e68e%5E...20629033923fbb7f7ac975355b702e797312e68e)

**Pull request merged**